### PR TITLE
Make current_peers empty when disabled

### PR DESCRIPTION
**Pull request by martinsumner**
https://github.com/nhs-riak/riak_kv/issues/9

### CODE REVIEW

**Comment by martinsumner** [comment]
https://github.com/nhs-riak/riak_test/pull/6

File: src/riak_kv_replrtq_peer.erl [hunk 1]
```diff
@@ -183,46 +183,48 @@ do_discovery(QueueName, PeerInfo, Type) ->
...
-            riak_kv_replrtq_snk:add_snkqueue(QueueName,
-                DiscoveredPeers,
-                SnkWorkerCount,
-                PerPeerLimit),
-            true
+        {DiscoveredPeers, count_change} ->
+            riak_kv_replrtq_snk:add_snkqueue(
+                QueueName, DiscoveredPeers, SnkWorkerCount, PerPeerLimit),
+            true;
+        {DiscoveredPeers, OldPeers} when is_list(OldPeers) ->
```
Comment (ThomasArts) [inline src/riak_kv_replrtq_peer.erl]: This is kind of weird to call CurrentPeers by name OldPeers
```suggestion
        {DiscoveredPeers, _} when is_list(CurrentPeers) ->
```
OldPeers and CurrentPeers are for sure the same, instead one wants to print the length of discovered peers, right?

File: src/riak_kv_replrtq_peer.erl [hunk 2]
```diff
@@ -183,46 +183,48 @@ do_discovery(QueueName, PeerInfo, Type) ->
...
+            lager:info(
+                "Type=~w discovery old_peers=~w new_peers=~w",
+                [Type, length(CurrentPeers), length(OldPeers)]),
```
Comment (ThomasArts) [inline src/riak_kv_replrtq_peer.erl]:
```suggestion
                [Type, length(CurrentPeers), length(DiscoveredPeers)]),
```

**Review by ThomasArts — COMMENTED** [review]
I think that OldPeers in match was a mistake and that you really want to print the discovered peers length... I might be wrong

**Review by ThomasArts — APPROVED** [review]

**Pull request merged**